src/app/visit/status.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+#include <string>
+
+enum class Status
+{
+    Canceled,
+    Relocated,
+    Confirmed,
+    Visited
+};
+
+std::string toString(Status status);
```

Comment:
[Asking myself and everyone else] 
Maybe it could be inline? 
No matter which case it will be 4 jumps in assembly at most